src/sysrepo.h

```diff
@@ -962,6 +962,15 @@ int sr_nacm_apply_changes(sr_session_ctx_t *session, uint32_t timeout_ms);
  */
 int sr_has_changes(sr_session_ctx_t *session);
 
+/**
+ * @brief Retrieve stored changes (prepared, not yet applied).
```

Comment:
Include a note somewhere that the pointer stops being valid as soon as the changes are applied or discarded.